working-migration-guides/README.md

```diff
@@ -0,0 +1,51 @@
+# Bevy's Release Notes Process
+
+Hi! Did someone add `M-Needs-Migration-Guide` to your PR? If so, you're in the right place.
+Let's talk about how this process works.
+
+When we make breaking changes to Bevy, we need to communicate them to users so their libraries and applications can be moved to the new Bevy version.
+To do this, we write and ship a migration guide for every major Bevy version.
```

Comment:
Similarly, we could link to <https://bevyengine.org/learn/migration-guides/introduction/> here.